src/components/PingChart.vue

```diff
@@ -60,6 +64,11 @@ export default {
     },
     computed: {
         chartOptions() {
+            const minuteTimeFormat = `${
+                this.use12HourTimeFormat ? "hh" : "HH"
+            }:mm${
+                this.use12HourTimeFormat ? " A" : ""
+            }`;
```

Comment:
nit: do you think extracting the first variable looks better?

```suggestion
            const hourTimeFormat = this.use12HourTimeFormat ? "hh" : "HH";
            const minuteTimeFormat = `${hourTimeFormat}:mm${
                this.use12HourTimeFormat ? " A" : ""
            }`;
```

Reply:
Yes, makes sense, thanks!